tutorials/tutorial-setting-up-timescale-cloud-endpoint-for-prometheus.md

```diff
@@ -90,15 +90,14 @@ service on Timescale Cloud!
 
 ### Next Steps
 
-Next, [create a monitoring system][monitoring-setup-tutorial] using Timescale, 
-Grafana, and Prometheus to ingest and analyze Prometheus metrics from your 
-Timescale Cloud instance.
+Next, [use Promscale][promscale] with Timescale, Grafana, and Prometheus to ingest 
+and analyze Prometheus metrics from your Timescale Cloud instance.
 
 
 [timescale-cloud]: https://www.timescale.com/products
 [timescale-cloud-install]: /getting-started/explore-cloud
 [get-prometheus]: https://prometheus.io
 [timescale-cloud-get-started]: /getting-started/exploring-cloud
-[monitoring-setup-tutorial]: /tutorials/tutorial-setup-timescale-prometheus
```

Comment:
This link is referenced in line 9 (now broken).  Can you edit the paragraph to remove the link or re-purpose it to reference Promscale?

Reply:
fixed.